packages/iobio-react-components/src/percent_box.tsx

```diff
@@ -19,8 +19,30 @@
  *
  */
 
-function IobioPercentBox({ percentKey, title, totalKey }: { percentKey: string; title?: string; totalKey: string }) {
-	return <iobio-percent-box percent-key={percentKey} title={title} total-key={totalKey} />;
+import { useEffect } from 'react';
+import { setElementStyles } from './utils';
+
+function IobioPercentBox({
+	percentKey,
+	label,
+	styles,
+	totalKey,
+}: {
+	percentKey: string;
+	label?: string;
+	styles?: string;
+	totalKey: string;
+}) {
+	useEffect(() => {
+		const selector = `iobio-percent-box[percent-key=${percentKey}][total-key=${totalKey}]`;
+		const element = document.querySelector(selector);
```

Comment:
react refs might be useful here:
- no need to worry about multiple elements 
  - `document.querySelector` will return one element but what if the selector accidentally target multiple elements)
- avoids searching the DOM for an element
- inbuilt React library code for targeting DOM elements 
  - react will handle updating val to null if element is not available
- doesn't depend on `document` api being available (eg. for SSR)